app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt

```diff
@@ -99,6 +102,7 @@ private val placeholderStory = PocketRecommendedStory("", "", "", "", "", 0, 0)
  * @param story The [PocketRecommendedStory] to be displayed.
  * @param onStoryClick Callback for when the user taps on this story.
  */
+@OptIn(ExperimentalComposeUiApi::class) // for testTagsAsResourceId
```

Comment:
```suggestion
@OptIn(ExperimentalComposeUiApi::class)
```
Doesn't seem necessary to add a comment to note which API is experimental. This practice wouldn't scale if we also used a number of other experimental compose UI API nor do I envision everyone doing this consistently if we kept this comment. So, I would prefer if we didn't add it to avoid inconsistence and debate down the line.